tests/conftest.py

```diff
@@ -299,8 +299,15 @@ def mechanical_pool():
 
 
 def pytest_addoption(parser):
+    mechanical_path = atp.get_mechanical_path(False)
```

Comment:
I wonder if it makes sense to run the tests at all if atp can't find the paths

Reply:
I think if atp can't find the path, then we could keep the default 232 option, but I see what you mean. If it can't find the path on a 241 machine, then it would be pointless to run 232 tests on it by default.